react/Identification.tsx

```diff
@@ -1,7 +1,174 @@
-import React from 'react'
+import classNames from 'classnames'
+import React, { useState, useEffect } from 'react'
+import { useLazyQuery, useMutation } from 'react-apollo'
+import { defineMessages, FormattedMessage, useIntl } from 'react-intl'
+import { IconCheck, Input, Button, Divider, Spinner } from 'vtex.styleguide'
+import { useChildBlock, ExtensionPoint, useRuntime } from 'vtex.render-runtime'
+import ProfileQuery from 'vtex.checkout-resources/QueryProfile'
+import UpdateOrderFormProfileMutation from 'vtex.checkout-resources/MutationUpdateOrderFormProfile'
```

Comment:
How about 
```suggestion
import { QueryProfile as ProfileQuery } from 'vtex.checkout-resources'
import { MutationUpdateOrderFormProfile as UpdateOrderFormProfileMutation } from 'vtex.checkout-resources'
```
and deleting `typings/vtex.checkout-resources.d.ts`?

Reply:
won't that result in the same issue we had of loading all the queries in runtime while using just a couple of them?

Reply:
/O\\
You are right. It is just so bad having that `d.ts` file but I guess we don't have a better solution for now.

Reply:
I don't really like maintaining that either, but that's a limitation in the package we use :-/